### PR TITLE
Fix Tests And Link

### PR DESCRIPTION
**Pull request by qwwqwwq**
- Shift to i18n address part names broke some tests.
- At some point net.getUsersValues was removed, which broke the ui-router link in the claim flow

### CODE REVIEW

File: winterfell/models/letter.js [hunk 1]
```diff
@@ -9,20 +9,20 @@ var letterSchema = new Schema({
...
     toAddress: {
         name: String, // Name line
-        line1: String,
-        line2: String,
+        street1: String,
+        street2: String,
         city: String,
-        state: String,
-        zip: String,
+        province: String,
```
Comment (ahsu1230) [inline winterfell/models/letter.js]: thanks

File: winterfell/webapp/src/js/net.js [hunk 2]
```diff
@@ -56,6 +56,9 @@ app.factory('net', ['xhrEnv', '$http', function(xhrEnv, $http) {
     getUserInfo: function() {
       return httpGet("/user");
     },
+    getUserValues: function() {
```
Comment (ahsu1230) [inline winterfell/webapp/src/js/net.js]: woops that was me. My bad, thought it was unused.

File: winterfell/webapp/src/js/profileCtrl.js [hunk 3]
```diff
@@ -15,32 +15,15 @@ app.controller('profileCtrl', ['$scope', '$location', '$window', 'Profile', 'cla
...
-        $scope.claims[i].id = $scope.claims[i].externalId; // todo, server shouldn't be sending rowId and externalId. We only reference claims through claim.id
+        $scope.claims[i].id = $scope.claims[i]._id;
         $scope.claims[i].header = createHeaderString($scope.claims[i]);
         $scope.claims[i].date = $filter('date')(new Date($scope.claims[i].stateUpdatedAt), 'MM/dd/yyyy');
       }
     }
     init();
 
     $scope.clickEditInfo = function() {
-      var newScope = $scope.$new(true);
```
Comment (ahsu1230) [inline winterfell/webapp/src/js/profileCtrl.js]: nooooooooooo

Comment (qwwqwwq) [inline winterfell/webapp/src/js/profileCtrl.js]: this seems like git error I dont know why I would have changed this, sorry will fix

**Comment by qwwqwwq** [comment]
Fixed the issue!

**Pull request merged**